### PR TITLE
Add BGL Corp

### PR DESCRIPTION
**Pull request by ryangcode**

### CODE REVIEW

File: _data/finance.yml
Comment (Carlgo11) [inline _data/finance.yml]:
```suggestion
      doc: https://sf360.zendesk.com/hc/en-au/articles/360020963051
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Is the 2FA solution only for Simple Fund 360?

File: _data/finance.yml
Comment (ryangcode) [reply _data/finance.yml]: Thanks for reviewing it.  The MFA is for all our products, not only for simple fund 360. The login part is handled by the same app and the process is the same for all products.

**Comment by ryangcode** [comment]
@Carlgo11 Hi, wonder if you could merge the changes. Thanks.

**Review by Carlgo11 — DISMISSED** [review]

**Comment by ryangcode** [comment]
@Carlgo11 Hi, seemed that at least two reviewers are required for it to be merged. Wonder if you know anyone else could help to review this pull request. Thanks.

**Comment by mxxcon** [comment]
@ruiyang does this 2fa apply only to SimpleFund360 or to CAS360 and BGL Client Centre and other sites too?

**Comment by ryangcode** [comment]
@mxxcon it is for SimpleFund360, CAS360 and BGL API. not for BGL Client Centre.

**Comment by mxxcon** [comment]
> @mxxcon it is for SimpleFund360, CAS360 and BGL API. not for BGL Client Centre.

Perhaps it should be mentioned in `exceptions:`?

**Review by mxxcon — APPROVED** [review]

**Comment by ryangcode** [comment]
@Carlgo11 Hi, I have made all required changes, pls have a look and approve this pull request. Thanks.

**Comment by ryangcode** [comment]
@Carlgo11 Hi, any feedback?

**Comment by ryangcode** [comment]
@mxxcon Hi, could you help to ask another repository owner to review and approve this pull request? It needs another pull request approval to be merged. @Carlgo11 seems on leave. Thanks.

**Comment by ryangcode** [comment]
@jdavis Hi, seems that you are one of the maintainers of this repo. Wonder if you could help to review this pull request and merge it. Thanks.

**Comment by ryangcode** [comment]
@Carlgo11 Seemed that you are busy with other things and no response after I responded to your review comment after 1 and a half month.

@jdavis I assume you are the main maintainer of this repo. given the current situation, I'll close the pull request and submit a new one if still no response in 7 days. Please let me know if any issues.

**Review by kmpoppe — APPROVED** [review]

**Pull request merged**
